src/test/java/ReversTest.java

```diff
@@ -0,0 +1,78 @@
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ReversTest {
+
+    @Test
+    public void keysEqualsValues() throws Exception {
+        List<Integer> keys = new ArrayList<>();
+        List<String> values = new ArrayList<>();
+        keys.add(1);
+        keys.add(2);
+        keys.add(3);
+        keys.add(4);
+        values.add("a");
+        values.add("b");
+        values.add("c");
+        values.add("d");
+        Revers.createMapOption1(keys, values).forEach((key, value) -> System.out.println(key + ":" + value));
+    }
+
+    @Test
+    public void keysMoreThenValues() {
```

Comment:
mistype in the word "than" (please rename some other methods as well)

Reply:
Renamed